executor/row.rs

```diff
@@ -30,7 +30,7 @@ impl<'a> Row<'a> {
         self.row.len()
     }
 
-    pub(crate) fn get(&self, position: VariablePosition) -> &VariableValue<'static> {
+    pub fn get(&self, position: VariablePosition) -> &VariableValue<'static> {
```

Comment:
Only needs to be pub because of an integration test. Should I introduce a `#[cfg(test)]` util method?

Reply:
i think it's ok